UI/forms/OBSBasic.ui

```diff
@@ -753,7 +753,7 @@
           <rect>
            <x>0</x>
            <y>0</y>
-           <width>71</width>
+           <width>64</width>
```

Comment:
Was this intentional? Seems odd to reduce the width when the smallest button is the one that was removed. Maybe I'm just ignorant on how Qt works in this regard.

Reply:
For some reason it always does this whenever I do change something in QT designer.